tools/amd_build/patches/a_aten_src_THCS_generic_THCSTensorMath.cu.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/aten/src/THCS/generic/THCSTensorMath.cu b/aten/src/THCS/generic/THCSTensorMath.cu
+index 319652fdf..181ab7634 100644
+--- a/aten/src/THCS/generic/THCSTensorMath.cu
++++ b/aten/src/THCS/generic/THCSTensorMath.cu
+@@ -173,13 +173,16 @@ void THCSTensor_(sspaddmm)(THCState *state, THCSTensor *r_, real beta, THCSTenso
+ }
+ 
+ void THCSTensor_(hspmm)(THCState *state, THCSTensor *r_, real alpha, THCSTensor *sparse_, THCTensor *dense) {
++#if defined(__HIP_PLATFORM_HCC__)
++  #define THRUST_EXEC(fn, ...) // whitespace
```

Comment:
This also looks like it will silently start returning incorrect results. Would be better to replace `// whitespace` with `throw ...`